src/assets/icons/Outside-Link.svg

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is an Icon called `assets/icon/External-Link.svg` you can use that one instead

Reply:
OK